util/util.go

```diff
@@ -148,3 +148,11 @@ func RemoveOrphanedInstances(verbose bool) error {
 
 	return nil
 }
+
+// AbsTarPathMatches tells whether the tar header name matches the path.
+// This function is needed since some tar files prefix its header names
+// with / and some not. NOTE: argument 'path' should start with '/' while
+// 'tarPath' should be header.Name.
+func AbsTarPathMatches(path, tarPath string) bool {
```

Comment:
You are right, the function really is quite specific for mpm package usecase. So I moved it in cmd/package.go.